Library/Homebrew/test/cmd/info_spec.rb

```diff
@@ -20,6 +20,13 @@
   end
 end
 
+describe "brew info", :integration_test, :needs_macos do
```

Comment:
Could this be in `Library/Homebrew/test/missing_formula_spec.rb` instead? Integration tests are dramatically slower so we want to avoid them where we can.